index.html

```diff
@@ -152,7 +152,9 @@
 
     <p>
 <a>Decentralized identifiers</a> (DIDs) are a new type of identifier to
-provide verifiable, decentralized digital identity. These new identifiers are
+provide verifiable, decentralized digital identity. A DID identifies an 
+entity that is capable of creating digital signatures, e.g. a natural person, 
```

Comment:
This is wrong. :(

A DID identifies an entity. That entity may or may not be capable of creating digital signatures... the verification relationships between the DID and verification methods establish which entities may create digital signatures... these verification methods may allow different entities to create different cryptographic proofs for different purposes. So, this isn't simply a one to one mapping. It's a many to one mapping... not all cryptographic proofs are digital signatures.

I suggest that @Oskar-van-Deventer and @jandrieu get together and come up with a new definition based on the discussion around the three types of entity that a DID Document can be associated with... I forget the issue where that's being discussed, but @jandrieu probably knows.

Reply:
Hi Manu,
What is your point? 
a) The PR is too narrowly scoped, and "digital signature" should be replaced with "cryptographic proof", or
b) It is perfectly fine to have a DID identify a cat or a river.
I can easily implement a). I see fundamental semantics and interoperability issues with b).
Oskar
P.S. Are you referring to issue #199?

Reply:
> What is your point?

My point is that there are multiple issues with the PR and we need more people to weigh in on it. :)

As an editor, I can't merge it because I don't think it has consensus. I'd like to see the people involved in #199 participate in crafting the text in this PR.

> a) The PR is too narrowly scoped, and "digital signature" should be replaced with "cryptographic proof", or

"digital signature" and "cryptographic proof" is probably too technical and low level as introductory text. We had previous used text like the following "interact with", "engage with". It was sharpened to be more precise, but I think we lost something there when we did that.

> b) It is perfectly fine to have a DID identify a cat or a river.

Yes, correct.

> I can easily implement a). I see fundamental semantics and interoperability issues with b).

"cryptographic proof" is more accurate, but is a step in the wrong direction... again, we need more people to weigh in on this. I would rather we go for something that's a bit more vague (but still correct) in the abstract and then further clarify in the introduction.

> P.S. Are you referring to issue #199?

Yes, thank you.

Reply:
Many, thank you. I updated my PR following your suggestions. I'll await feedback from the experts, like @jandrieu. Oskar